### PR TITLE
[WIP, WOC] #3427 / ETHDenver2019 Event Commit

### PR DESCRIPTION
**Pull request by kuhnchris**
##### Description

This PR adds the ETHDenver2019 event page(s)
** CURRENTLY WORK IN PROGRESS, DO NOT ATTEMPT TO MERGE YET! **

##### Checklist

- [x] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [x] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Affected core subsystem(s)

ui, db

##### Refers/Fixes

Fixes: #3427 

##### Testing and Sign-off

image

###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

**Comment by owocki** [comment]
just pulled this down on my local to test!  looks like its coming together.. let me know when its ready for QA

**Comment by kuhnchris** [comment]
yeah I have to test it first locally, I only did a hard coding session yesterday night to get everything rolling, but now I have to test it. I had to do a... work around for the funder-reply comment, since we do not use IPFS there, so I had to add an extra call to store the comment... + ensure that we only can store 1 comment there and not have some random bot fill up our database with random crap.

**Comment by owocki** [comment]
making some changes to this PR over at https://github.com/gitcoinco/web/pull/3707

File: app/event_ethdenver2019/views.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    kudos_row = []
+    kudos_selected = Event_ETHDenver2019_Customizing_Kudos.objects.filter(active=True, final=False).all()
+    all_kudos_collected = True
+
+    for kudos in kudos_selected:
+        kudos_obj = {
+            "kudos": kudos.kudos_required,
+            "received": False,
+            "customizing": kudos,
+            # "expanded_kudos": vars(kudos.kudos_required)
```
Comment (owocki) [inline app/event_ethdenver2019/views.py]: should this be commented out?

File: app/event_ethdenver2019/views.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+from dashboard.utils import get_nonce, get_web3
+from gas.utils import recommend_min_gas_price_to_confirm_in_time
+from kudos.utils import kudos_abi
+from ratelimit.decorators import ratelimit
+from retail.helpers import get_ip
+from web3 import Web3
+from kudos.models import BulkTransferCoupon, BulkTransferRedemption, KudosTransfer, Token
+
+'''
+def ethdenver2019_redeem(request):
```
Comment (owocki) [inline app/event_ethdenver2019/views.py]: @kuhnchris should this be commented out?

**Pull request merged**